docs/markdown/main/index.md

```diff
@@ -1,7 +1,7 @@
 <div class="hyHomePage">
     <header class="main-header hyHomeHeaderMain">
         <div class="hyHomeHeader text-center">
-            <h1>SAP Hybris Techné</h1>
+            <h1>SAP Hybris Techné <sub>v1.5.4</sub></h1>
```

Comment:
should we move this version number below the download button? I checked some libs (bootstrap, jquery, angualr, etc.) quickly and they all have the version number on or near the download button